### ISSUE TITLE
Issues while building Tinodios

### ISSUE THREAD

**Issue by Kuvonchbeky** [tinode/ios#112]
1. Header 'TinodeSDk-Swift.h' not found on module.modulemap. line: 9
2. Could not build Objective-C module 'TinodeSDK' on BaseDb.swift. line: 10
3. Missing return in a function expected to return '[TinodeEventListener]' on Tinode.swift. line: 212
XCode version: 10.3

**Comment by or-else** [comment]
Something is wrong with your build setup. The code is complete and compiles without any issues with XCode Version 11.1 (11A1027):

```
ios gene$ git remote -v
origin	https://github.com/tinode/ios (fetch)
origin	https://github.com/tinode/ios (push)
```

```
ios gene$ git branch
  devel
  master
* more-fixes
```

```
ios gene$ git status
On branch more-fixes
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	Tinodios.xcodeproj/xcshareddata/
	Tinodios/Gene.developerprofile
	sqlios.sh

nothing added to commit but untracked files present (use "git add" to track)
```

Build output:
```
Pods project Group
Localization Group
/Users/gene/ventures/tinode/ios/Pods/Pods.xcodeproj Enable Base Internationalization
TinodiosDB Group
Swift Compiler Warning Group
/Users/gene/ventures/tinode/ios/TinodiosDB/BaseDb.swift:159:64: Treating a forced downcast to 'Int64' as optional will never produce 'nil'
/Users/gene/ventures/tinode/ios/TinodiosDB/BaseDb.swift:159:66: Use 'as?' to perform a conditional downcast to 'Int64'
/Users/gene/ventures/tinode/ios/TinodiosDB/BaseDb.swift:159:34: Add parentheses around the cast to silence this warning
/Users/gene/ventures/tinode/ios/TinodiosDB/SqlStore.swift:27:36: Result of call to 'saveDeviceToken(token:)' is unused
TinodeSDK project Group
Localization Group
/Users/gene/ventures/tinode/ios/TinodeSDK.xcodeproj Enable Base Internationalization
Tinodios Group
Swift Compiler Warning Group
/Users/gene/ventures/tinode/ios/Pods/Firebase/CoreOnly/Sources/Firebase.h:57:10: "FirebaseAnalytics.framework is not included in your target. Please add `Firebase/Analytics` to your Podfile or add FirebaseAnalytics.framework to your project to ensure Firebase Messaging works as intended."
```

**Comment by Kuvonchbeky** [comment]
solved by updating to XCode 11.2.1 (11B500). Thanks
